src/utils/renderer-process-directory.ts

```diff
@@ -19,39 +19,36 @@
 import * as assert from 'assert';
 import * as electron from 'electron';
 import * as fs from 'fs-extra';
-import { resolve } from 'path';
+import * as path from 'path';
 
 const app = electron.app || electron.remote.app;
 const userDataPath = app.getPath('userData');
-const rendererPattern = /^renderer-\d+$/;
+const rendererPattern = /^renderer-.+$/;
 
-// Call this with `process.pid.toString()`
-export const getRendererProcessDirectory = (pid: number) => {
-  const pidString = pid.toString();
-  return resolve(userDataPath, `renderer-${pidString}`);
-};
-
-export const getRendererProcessDirectories = () => {
-  const directories = fs.readdirSync(userDataPath);
-  return directories
-    .filter(name => rendererPattern.test(name))
-    .map(name => resolve(userDataPath, name));
-};
-
-export const changeRendererProcessDirectory = () => {
+export const changeRendererProcessDirectory = (type: string) => {
```

Comment:
I'm not sure I understand the rationale for creating a cache directory for each window?

Reply:
We can't have multiple processes accessing the same file/sync session. This is problematic because each renderer runs in its own process, and the server administration window opens the admin Realm. This means that if you open the admin Realm in a browser window, you'll get a realm-js crash.

Reply:
Ah okay. I thought that everything ran in the same process/thread :+1: